app/serializers/organization_page_serializer.rb

```diff
@@ -0,0 +1,36 @@
+class OrganizationPageSerializer
+  include RestPack::Serializer
+  include CachedSerializer
+
+  attributes :id, :href, :created_at, :updated_at, :url_key, :title,
+    :language, :content, :type
+
+  can_include :organization
+  can_filter_by :url_key, :language
+
+  def self.page_href(page, options)
```

Comment:
That's true for a lot of this, as I said up top. Obviously it'd be better if all the duped code was cleaned up. This was faster, and short of adding another level of organization here we won't need it again. I can extract this all out for the serializers, controllers, etc., if you'd prefer.